peekingduck/pipeline/nodes/model/yoloxv1/yolox_files/detector.py

```diff
@@ -70,8 +72,7 @@ def predict_object_bbox_from_image(
 
         Args:
             image (np.ndarray): Input image.
-            class_names (List[str]): List of human-friendly object category
-                names.
+            detect_ids (List[int]): list of label ids to be detected
```

Comment:
```suggestion
            detect_ids (List[int]): List of label IDs to be detected.
```

Reply:
Updated comment (similarly for Yolo `detector.py`).